draft-ietf-quic-multipath.md

```diff
@@ -541,9 +541,14 @@ send path control information and specifically acknowledge packets belonging to
 packet number space.
 
 A path in the "Validating" state performs path validation as described
-in {{Section 8.2 of QUIC-TRANSPORT}}. An endhost should not send
+in {{Section 8.2 of QUIC-TRANSPORT}}. An endhost SHOULD NOT send
 non-probing frames on a path in "Validating" state, as it has no
-guarantee that packets will actually reach the peer.
+guarantee that packets will actually reach the peer, with an exception
+of the ACK_MP frame. An endhost MAY bundle the ACK_MP frame with the
+PATH_RESPONSE frame; in this case it is RECOMMENDED to consider
+the acknowledgement information as opportunistic and repeat
+the acknowledged packet ranges in the next ACK_MP frame.
```

Comment:
Not sure if this needs to be called out explicitly. ACKs are always opportunistic in QUIC, as ACK frames are not ack-eliciting.

Section 13.2.4 of RFC 9000 defines how to prune old ACK ranges, and I don't think we need to do anything special here.